### ISSUE TITLE
Some API's are missing

### ISSUE THREAD

**Issue by franknesse** [hubspot-net/HubSpot.NET#64]
Some API's are not available. Eg. HubSpotTimelineApi and HubSpotEmailEventsApi
Are these not supported in the .Net 4.6.1 version?

**Comment by Psypher9** [comment]
> @franknesse, unfortunately these are not available in the .NET 4.6.1 version. Those APIs will be part of our 1.0 release targeting .NET 4.6.1 and .NET Standard 2.0
<sub>[View in Slack](https://hubspotnet.slack.com/archives/CD965LVJ7/p1574870836000300)</sub>

**Comment by franknesse** [comment]
Ok, thanks. You mean , they *will* be available in 4.6.2 in version 1.0? If so, when is version 1.0 planned to be released? .Net Standard 2.0 is not an option for me yet.

**Comment by Psypher9** [comment]
Yes, they will be available for 4.6.1. We are targeting both frameworks at the moment. We don't have a set date yet. I need to talk more with the team and set some testing goals, because we are not confident that the 1.0 release is fully stable yet.

I'd welcome the help the test if you'd be up for it!

**Comment by franknesse** [comment]
If there is any possibility to play around with the 1.0 pre-release I would be happy to try and send you feedback in case I run into bugs or other issues. Structural testing is not an option however, since my experience with hubspot is very limited as well as my time to spend on it.

**Comment by Vdm88** [comment]
Forms APi ? missing

**Comment by KennyKaplanADA** [comment]
Missing Merge for Contacts

**Comment by Psypher9** [comment]
@Vdm88 @KennyKaplanADA If you find an API missing, please open a new issue specifically for that API, thank you!
